### PR TITLE
Remove ShellJS copy and use the NPM version

### PR DESCRIPTION
**Pull request by timvandermeij**
There is no need to have a copy of ShellJS in the repository as it is also available on NPM. The NPM version is also much newer. This way we do not have to update this anymore and let NPM do that automatically.

### CODE REVIEW

File: make.js [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
 'use strict';
 
-require('./external/shelljs/make');
+require('shelljs/make');
```
Comment (Snuffleupagus) [inline make.js]: I'm wondering if it would be a good idea to enclose this in a `try... catch` block, e.g. something like this:
```javascript
try {
  require('shelljs/make');
} catch (ex) {
  console.log('Run "npm install"'); // Probably needs a better message
  return;
}
```
When I tested the patch the first time, running any of the `make` commands just threw an error, and it took me a couple of seconds to realise that I needed to run `npm install`.
For less experienced users, it's not unlikely that they *could* get stuck on this, so a message telling them what to do seems like a good idea to me.

*Edit:* A somewhat fun fact is that if the above is implemented, I think that it would actually make PR #5908 obsolete :-)

Comment (timvandermeij) [inline make.js]: Done. Great catch, thanks for noticing this!

**Comment by Snuffleupagus** [comment]
This does look perfectly fine to me, and it'd be great to be able to get rid of all this code from the repo!
However before merging this, I want to check with Brendan/Yury on IRC, to make sure that there are not any weird dependencies in edge cases (I'm mostly concerned about the bots).

/botio test

**Comment by Snuffleupagus** [comment]
Thank you!

**Comment by yurydelendik** [comment]
@timvandermeij we probably need to fix version number to be exact to avoid changes similar to https://github.com/arturadib/shelljs/pull/207
